src/index.html

```diff
@@ -10,8 +10,164 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/   create-application"
+      class="form">
+
+      <fieldset  class="field-form" name="personalInformation">
+        <legend>Personal information</legend>
+        <label class="input-form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-form">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-form">
+         How old are You?:
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="input-form">
+          Fool date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
```

Comment:
the tag should be written in one line if there are less than 3 attributes
```suggestion
          <input type="date" name="dateOfBirth">
```

Reply:
thankyou  I will fix it now